2021-10-17 22:15:17,276 INFO    MainThread:412751 [wandb_setup.py:_flush():69] setting env: {}
2021-10-17 22:15:17,276 INFO    MainThread:412751 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-17 22:15:17,276 INFO    MainThread:412751 [wandb_init.py:_log_setup():342] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Intel-Image-Classification-V2/wandb/run-20211017_221517-bdsmpxmo/logs/debug.log
2021-10-17 22:15:17,276 INFO    MainThread:412751 [wandb_init.py:_log_setup():343] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Intel-Image-Classification-V2/wandb/run-20211017_221517-bdsmpxmo/logs/debug-internal.log
2021-10-17 22:15:17,276 INFO    MainThread:412751 [wandb_init.py:init():375] calling init triggers
2021-10-17 22:15:17,276 INFO    MainThread:412751 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {}
2021-10-17 22:15:17,276 INFO    MainThread:412751 [wandb_init.py:init():397] re-initializing run, found existing run on stack: 2snzs8gh
2021-10-17 22:15:17,277 INFO    MainThread:412751 [wandb_run.py:finish():1271] finishing run ranuga-d/Intel-Image-Classification-V2/2snzs8gh
2021-10-17 22:15:17,281 INFO    MainThread:412751 [jupyter.py:save_history():382] saving 25 cells to _session_history.ipynb
2021-10-17 22:15:17,287 INFO    MainThread:412751 [wandb_run.py:_config_callback():843] config_cb ('_wandb', 'session_history') code/_session_history.ipynb None
2021-10-17 22:15:17,302 INFO    MainThread:412751 [jupyter.py:save_ipynb():301] looking for notebook: 00.ipynb
2021-10-17 22:15:17,863 INFO    MainThread:412751 [wandb_init.py:_jupyter_teardown():274] saved code and history
2021-10-17 22:15:17,863 INFO    MainThread:412751 [wandb_init.py:_jupyter_teardown():275] cleaning up jupyter logic
2021-10-17 22:15:17,863 INFO    MainThread:412751 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-10-17 22:15:17,863 INFO    MainThread:412751 [wandb_run.py:_restore():1543] restore
2021-10-17 22:15:22,869 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: None
2021-10-17 22:15:27,926 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 2
  artifact_count: 4
}
pusher_stats {
  uploaded_bytes: 348997
  total_bytes: 348997
}

2021-10-17 22:15:28,529 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 2
  artifact_count: 4
}
pusher_stats {
  uploaded_bytes: 348997
  total_bytes: 348997
}

2021-10-17 22:15:28,903 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 348997
  total_bytes: 386313
}

2021-10-17 22:15:29,005 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 348997
  total_bytes: 386313
}

2021-10-17 22:15:29,107 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 348997
  total_bytes: 386313
}

2021-10-17 22:15:29,208 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 348997
  total_bytes: 386313
}

2021-10-17 22:15:29,310 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 348997
  total_bytes: 386313
}

2021-10-17 22:15:29,414 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 348997
  total_bytes: 386313
}

2021-10-17 22:15:29,516 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 348997
  total_bytes: 386313
}

2021-10-17 22:15:29,618 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 348997
  total_bytes: 386313
}

2021-10-17 22:15:29,719 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 380578
  total_bytes: 386313
}

2021-10-17 22:15:29,823 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 386287
  total_bytes: 386313
}

2021-10-17 22:15:29,925 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 386313
  total_bytes: 386313
}

2021-10-17 22:15:30,028 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 386313
  total_bytes: 386313
}

2021-10-17 22:15:30,129 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 386313
  total_bytes: 386313
}

2021-10-17 22:15:30,231 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 386313
  total_bytes: 386313
}

2021-10-17 22:15:30,333 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 386313
  total_bytes: 386313
}

2021-10-17 22:15:30,434 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 386313
  total_bytes: 386313
}

2021-10-17 22:15:30,795 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 386313
  total_bytes: 386313
}

2021-10-17 22:15:30,897 INFO    MainThread:412751 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 7
  artifact_count: 4
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 386313
  total_bytes: 386313
}

2021-10-17 22:15:32,011 INFO    MainThread:412751 [wandb_run.py:_show_files():1915] logging synced files
2021-10-17 22:15:32,015 INFO    MainThread:412751 [wandb_init.py:init():424] starting backend
2021-10-17 22:15:32,015 INFO    MainThread:412751 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-17 22:15:32,018 INFO    MainThread:412751 [backend.py:ensure_launched():135] starting backend process...
2021-10-17 22:15:32,069 INFO    MainThread:412751 [backend.py:ensure_launched():140] started backend process with pid: 421971
2021-10-17 22:15:32,069 INFO    MainThread:412751 [wandb_init.py:init():429] backend started and connected
2021-10-17 22:15:32,080 INFO    MainThread:412751 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-10-17 22:15:32,082 INFO    MainThread:412751 [wandb_init.py:init():477] updated telemetry
2021-10-17 22:15:32,082 INFO    MainThread:412751 [wandb_init.py:init():500] communicating current version
2021-10-17 22:15:32,855 INFO    MainThread:412751 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-17 22:15:32,856 INFO    MainThread:412751 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-17 22:15:33,475 INFO    MainThread:412751 [wandb_init.py:init():540] starting run threads in backend
2021-10-17 22:15:38,482 INFO    MainThread:412751 [wandb_run.py:_console_start():1601] atexit reg
2021-10-17 22:15:38,482 INFO    MainThread:412751 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-10-17 22:15:38,483 INFO    MainThread:412751 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-10-17 22:15:38,483 INFO    MainThread:412751 [wandb_run.py:_redirect():1536] Redirects installed.
2021-10-17 22:15:38,483 INFO    MainThread:412751 [wandb_init.py:init():565] run started, returning control to user process
2021-10-17 22:15:49,906 INFO    MainThread:412751 [wandb_setup.py:_flush():69] setting env: {}
2021-10-17 22:15:49,906 INFO    MainThread:412751 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-17 22:15:49,907 INFO    MainThread:412751 [wandb_init.py:_log_setup():342] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Intel-Image-Classification-V2/wandb/run-20211017_221549-1qmdoxpl/logs/debug.log
2021-10-17 22:15:49,907 INFO    MainThread:412751 [wandb_init.py:_log_setup():343] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Intel-Image-Classification-V2/wandb/run-20211017_221549-1qmdoxpl/logs/debug-internal.log
2021-10-17 22:15:49,908 INFO    MainThread:412751 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f70c2f3ef50>
2021-10-17 22:15:49,908 INFO    MainThread:412751 [wandb_init.py:init():375] calling init triggers
2021-10-17 22:15:49,908 INFO    MainThread:412751 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {}
2021-10-17 22:15:49,908 INFO    MainThread:412751 [wandb_init.py:init():397] re-initializing run, found existing run on stack: bdsmpxmo
2021-10-17 22:15:49,909 INFO    MainThread:412751 [wandb_run.py:finish():1271] finishing run ranuga-d/Intel-Image-Classification-V2/bdsmpxmo
